modules/models/src/main/java/org/eclipse/xpanse/modules/models/view/RegisteredServiceUserVo.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+
+package org.eclipse.xpanse.modules.models.view;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.eclipse.xpanse.modules.models.enums.Category;
+import org.eclipse.xpanse.modules.models.enums.Csp;
+import org.eclipse.xpanse.modules.models.enums.ServiceState;
+import org.eclipse.xpanse.modules.models.resource.Billing;
+import org.eclipse.xpanse.modules.models.resource.DeployVariable;
+import org.eclipse.xpanse.modules.models.resource.Flavor;
+import org.eclipse.xpanse.modules.models.resource.Region;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * Defines view object for end-user query registered service.
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class RegisteredServiceUserVo  extends RepresentationModel<RegisteredServiceUserVo> {
```

Comment:
We can avoid using the word "Register" when the end user comes into picture
May be we can name it something like "UserDeployableServiceVo"